core/services/keystore/keys/ocr2key/key_bundle.go

```diff
@@ -14,12 +14,20 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/store/models"
 )
 
+type OCR3SignerVerifier interface {
```

Comment:
@samsondav How does this get used in the code base? Not wild about the Sign3/Verify3 (and even less about the fact that this is located in `ocr2key`)

Also IIRC the Sign3/Verify3 isn't the interface that OCR3 expects is it? So do you need an adapter somewhere?

Reply:
Tried renaming to `ocr2plus` but its 1,000s of changes and makes the salient points of this PR unreadable.

Not to mention we use OCR2 all over the codebase to refer to both OCR2 and OCR3. There is probably a larger ticket to clean up that naming.